bundles/org.openhab.ui.basic/web-src/smarthome.js

```diff
@@ -1111,7 +1112,7 @@
 						(Math.PI * 2 - Math.atan(offsetY / offsetX) - Math.PI / 2) / (Math.PI * 2),
 				hsv = {
 					h: isNaN(angle) ? 0 : angle,
```

Comment:
And if you use 'Math.atan2()' it also eliminates the need for the NaN() check below, since the new code always produces a valid angle number.

Reply:
And, I forgot to mention: the old angle hue code requires the modulo 1.0 correction, whereas my new Math.atan2 code produces a result naturally in the range 0.0 .. 1.0 ..

Reply:
Done and tested